confluent-kafka-connect-jdbc.spec.in

```diff
@@ -48,10 +48,13 @@ exit 0
 
 %files
 %defattr(-,root,root)
-/etc/kafka-connect-jdbc
+
+%doc
 /usr/share/java/kafka-connect-jdbc
```

Comment:
why is /usr/share/java being classified as doc? it's jars in there, no?

Reply:
Looking at http://ftp.rpm.org/max-rpm/s1-rpm-inside-files-list-directives.html , I am not sure if this is doing as intended. %doc needs a list of files, but since you are providing none, then nothing is marked as docs. So the two dirs /usr/share/java and /ur/share/doc would just be normal packaged files, effectively no op.

To indicate an entire dir as docs, %docdir needs to be used that specifies the dir where the dir is also listed under %files. 

Can you clarify/confirm what's marked as normal file vs docs using `rpm -qlp` for normal files and `rpm -qdp` for docs? See full example in the rpm.org link.